3-0-java-core/src/test/java/com/bobocode/file_stats/FileStatsTest.java

```diff
@@ -0,0 +1,61 @@
+package com.bobocode.file_stats;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FileStatsTest {
```

Comment:
@maxstasiuk92 please follow this guide https://bobocode.atlassian.net/l/c/MeYPQPHx and add order.